client/components/NavBar/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Logo from './Logo/Logo';
+import HomeIcon from './HomeIcon/HomeIcon';
+import LifestyleIcon from './LifestyleIcon/LifestyleIcon';
+import ShopIcon from './ShopIcon/ShopIcon';
+import Login from './Login/Login';
+
+const NavBar = () => {
+  useEffect(() => {
```

Comment:
I think we can remove the useEffect here!

Reply:
Thank you. Removed those lines.